packages/zoslogs/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@zowe/zos-logs-for-zowe-sdk",
+  "version": "6.33.3",
+  "description": "Zowe SDK to interact with the z/OS logs",
+  "author": "Broadcom",
+  "license": "EPL-2.0",
+  "homepage": "https://github.com/zowe/zowe-cli/tree/master/packages/zoslogs#readme",
+  "bugs": {
+    "url": "https://github.com/zowe/zowe-cli/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zowe/zowe-cli.git"
+  },
+  "keywords": [
+    "zosmf",
+    "mainframe",
+    "CLI",
+    "zos",
+    "logs",
+    "z/OSMF",
+    "mvs",
+    "os390",
+    "z/OS",
+    "zowe"
+  ],
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "registry": "https://zowe.jfrog.io/zowe/api/npm/npm-local-release/"
+  },
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "build": "tsc --pretty",
+    "watch": "tsc --pretty --watch",
+    "clean": "rimraf lib",
+    "lint": "npm run lint:packages && npm run lint:tests",
+    "lint:packages": "tslint --format stylish -c ../../tslint-packages.json \"src/**/*.ts\"",
+    "lint:tests": "tslint --format stylish -c ../../tslint-tests.json \"__tests__/**/*.ts\"",
```

Comment:
Please update these scripts to call ESLint instead of TSLint. You can copy the NPM lint scripts from another SDK. 🙂